src/components/pages/coach/GenerateReport/index.jsx

```diff
@@ -168,26 +173,38 @@ class GenerateReport extends React.Component{
     fetchUserData = async () => {
         await this.props.client.query({
             query: GET_USERDATA,
-            variables: { "token": localStorage.getItem("fprint"), "id": this.state.user.id }
+            variables: { "token": localStorage.getItem("fprint"), "id": this.state.uid }
         }).then(({data}) => {
+            console.log(data);
+            console.log(data.anLatestByUid);
+
             if(data.anLatestByUid !== undefined){
-                this.setState({
-                    userdata: data.anLatestByUid
-                });
+                if(data.anLatestByUid.user.anamneseSet.length >= 1){
+                    this.setState({
+                        user: data.anLatestByUid.user,
+                        userdata: data.anLatestByUid,
+                        hasReports: true,
+                    }, () => this.fetchTemplate());
+                }
             }
         })
         .catch(error => {
+            if(error.message === "GraphQL error: User matching query does not exist."){
+                this.setState({
+                    exists: false
+                });
+            }
+            if(error.message === "GraphQL error: Anamnese matching query does not exist."){
+                this.setState({
+                    hasReports: false
+                });
+            }
             console.error("Query error:",error);
```

Comment:
# please remove `console` (no-console)

In JavaScript that is designed to be executed in the browser, it's considered a best practice to avoid using methods on `console`. Such messages are considered to be for debugging purposes and therefore not suitable to ship to the client. In general, calls using `console` should be stripped before being pushed to production.

```js
console.log("Made it here.");
console.error("That shouldn't have happened.");
```

Examples of **incorrect** code for this rule:

```js
/* eslint no-console: "error" */

console.log("Log a debug level message.");
console.warn("Log a warn level message.");
console.error("Log an error level message.");
```

Reply:
Noted.
However, the console functions do not exist without reason. It is a really great, fully integrated development tool.